components/divider/style/index.less

```diff
@@ -45,77 +47,43 @@
       transform: translateY(50%);
     }
   }
-  &-inner-text {
-    display: inline-block;
-    padding: 0 24px;
-  }
-  &-horizontal&-with-text-left {
-    display: table;
-    white-space: nowrap;
-    text-align: center;
-    background: transparent;
-    font-weight: 500;
-    color: @heading-color;
+
+  &-horizontal&-with-text-left,
+  &-horizontal&-with-text-right {
     font-size: @font-size-base;
-    margin: 16px 0;
+    .@{divider-prefix-cls}-inner-text {
```

Comment:
Plz note this change, it is come from:
```
&-horizontal&-with-text-left {
    &-inner-text {	
        display: inline-block;	
        padding: 0 10px;	
    }
}
```
I think it is not by design, the original css class will be expanded to `.ant-divider-with-text-left-inner-text `, and it is unused.

This requested DRY patch also fixed it to `.ant-divider-with-text-left .ant-divider-inner-text`, shrinks the padding when `orientation` is setted.